### PR TITLE
Replaces the mutagen biogenerator recipe with mutagen crates orderable from cargo

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

while chemistry can also make mutagen they generally  can't/don't want to constantly be making more and more of it throughout a round so cargo is able to order 200 units of it for 300 credits

### Why is this change good for the game?
more department interaction probably good
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
botany can no longer create mutagen at a biogenerator, but they can now have it ordered by cargo for 300 credits per 200 units
### What should players be aware of when it comes to the changes your PR is implementing?
do bounties
### What general grouping does this PR fall under? 
botany/cargo crates

# Changelog

:cl:  
rscdel: botany can no longer create mutagen with a biogenerator, you'll need to either synthesize it yourself or get it elsewhere
rscadd: cargo can now order crates with 200 units of mutagen in them for 300 credits
/:cl:

### CODE REVIEW

**Comment by LazennG** [comment]
All of these PR's encouraging interdepartmental interaction are good at making cargo something other  than 'assistants with maint access and a clubhouse'

**Comment by Jarod1200** [comment]
Interesting but also bruh

**Comment by TheLinuxOS** [comment]
I mean I like the thought but given that I typically go through 200 units to mutate a single plant sometimes this is horrible in practice

**Comment by Sniblet** [comment]
botany: "cargo can i get a mutagen crate i need to make ambrosia gaia"
botany: "cargo can i get a mutagen crate i need to make ambrosia gaia"
cargo: "wrfg is a mutagen carte talkt ro my cowotker ((it smy firsty game)("
botany: "cargo can i get a mutagen crate i need to make ambrosia gaia"
**botany has been trying to contact cargo for 5 minutes at this point**
cargo: "ok here comes 200u of mutagen for you just uh wait for the next... 10 minutes while we gather some easy bounties, send the shuttle, make the order, hold out for other orders, don't get any, eventually relent and call back the shuttle, and then our mulebot crawls over there"
**20 minutes pass**
botany: "cargo my vulgaris died while i was waiting is that mutagen coming along"
botany: "oh i see your mulebot's pathfinding broke in maint i got the crate"
**botany plants a new vulgaris plant, empties the entire crate into the plant tray, predictably it mutates to deus and no farther**
botany: "cargo can i get another mutagen crate"
botany: "cargo can i get like 4 mutagen crates i need to make ambrosia gaia"
cargo: "sorry this cargo tech is a rev this is the cmo using his pda from the brig"
botany: "ok can you give me mutagen cmo"
cmo: "no busy ask cargo"
**cmo killed by revs, round ends**

departmental interaction is probably good, except when you gate a department's entire fun factor behind other people giving them charity
mining's reliance on science is fine because everyone needs mining - science helps them to help themselves
robotics' reliance on mining is _usually_ fine because mining almost always happens
chemistry's reliance on holy water is fine because they can live without SR
botany drives its players to suicide without mutagen and cargo and chemistry both have next to no interests with them

**Comment by TheCalmingSurgeon** [comment]
As most my hours come from botanist I will say that I am biased towards my baby not being changed from the get go, it is not why I think this is bad however.

I'll address the good for the game bit first. I think bounties is a fine enough amount of basic department interaction between botany without getting into botany ordering seeds or bees throughout the round. If both sides wish to do them it leaves cargo with some creds and botany actually doing something and not being bored, if one side doesn't then oh well cargo can get money some place else and the botanists get their fun from smoking weed or whatever the fuck. Changing it so that cargo holds power over the botanists by taking the last worthwhile chem able to be made without a larger time investment makes it less fun for botanists and cargo can be all haha do our bounties or no mutagen (this scenario is going off what you say to begin with by botanists going to cargo and not chem for mutagen). Also the fact that some botany bounties, or all, need to be the mutated version of their species and so they are more reliant on cargo to even help in the first place. I can say that from my experience with cargo as botanist this PR would make me hate cargo even more now that I rely on them, every job should not rely on another but should have opportunity to come together and do stuff.

Second point would be that botanists will just go to chem round start since it's closest and the recipe is easy enough to hook up a good amount in a few seconds, or worse botanists grey tide in for mutagen and maybe even take the whole damn dispenser. I already see botanists that don't know we are different from other codebases and just go to chem cause that is the norm so making cargo able to give us mutagen is moot. Another reason not to change is that we are trying to be somewhat different from other codebases, yes? Making botany rely on other departments is just fitting in to how botany works on other stations I've played in, I like the uniqueness of yogs how I am able to perform my job better through using tools my job gives me.

Lastly would probably be the weakest point and that if everyone was completely jacked in and optimal then cargo would be the second fastest method of mutagen generation with actually playing botany being behind it with having to grow biomass in a large enough capacity to be worth a damn. It takes 600 for 30 units and my usual go to watermelons get me say 120 units, possibly another bottle so 150 in that case and it takes a few minutes. Botanists that use grass or soy get the fuck away from me. It may seem strange for me to ask that my slower method stays but again it really just comes down to me wanting to be self-reliant which I deem funner than haggle rp with a new techie or talking to a methhead chemist.

**Comment by ChesterTheCheesy** [comment]
You want botany stealing chem dispenser? Because that's how you get botany to steal a chem dispenser every round.
